change/@itwin-property-grid-react-6141cefa-e559-437e-85ff-3904f0fd207e.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "Add extract api step",
```

Comment:
```suggestion
  "comment": "",
```

Reply:
FYI, there is no need to make comment empty string. If type is set to `none` it won't impact `CHANGELOG.md` generation.